docs/content/doc/help/faq.en-us.md

```diff
@@ -275,4 +277,16 @@ By default, your LFS token will expire after 20 minutes. If you have a slow conn
 You may want to set this value to `60m` or `120m`.
 
 ## How can I create users before starting Gitea
-Gitea provides a sub-command `gitea migrate` to initialize the database, after which you can use the [admin CLI commands]({{< relref "doc/usage/command-line.en-us.md" >}}) to add users like normal.
+Gitea provides a sub-command `gitea migrate` to initialize the database, after which you can use the [admin CLI commands]({{< relref "doc/usage/command-line.en-us.md#admin" >}}) to add users like normal.
+
+## How can I enable password reset
+There is no setting for password resets. It is considered "enabled" when a [mail service]({{< relref "doc/usage/email-setup.md" >}}) is configured, and "disabled" otherwise.
```

Comment:
```suggestion
There is no setting for password resets. It is enabled when a [mail service]({{< relref "doc/usage/email-setup.md" >}}) is configured, and disabled otherwise.
```

Reply:
Manually merged change